.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+# Pull Request 
+
+## Proposed Changes
+Give a summary of the changes made in the repository and the issues fixed. List the dependencies used. Also mention the issue number.
```

Comment:
Hey @Shreesh90 untold open source rule xD solve one issue with every pr so this might give contributors the wrong idea of solving multiple issues with a single pr :sweat_smile: